templates/autoEdits/result.html.twig

```diff
@@ -216,7 +216,7 @@
                     </div>
                 {% endif %}
                 <p style="clear:both">
-                    <a target="_blank" href="https://phabricator.wikimedia.org/maniphest/task/create/?title=Request to add tool to AutoEdits&amp;projects=XTools&amp;description=Project: {{ project.domain }}%0A%0A{{ msg('auto-edits-request-name') }}:%0A%0A{{ msg('auto-edits-request-examples') }}:%0A%0A{{ msg('auto-edits-request-doc') }}:">
+                    <a target="_blank" href="https://meta.wikimedia.org/w/index.php?title=MediaWiki_talk:XTools-AutoEdits.json&action=edit&section=new&preload=MediaWiki_talk:XTools-AutoEdits.json/Preload">
```

Comment:
Is there also a link to some documentation about how it is now on meta?

Reply:
The thought is requests will mostly come from non-engineers, so I didn't want to confuse them with all the documentation. All maintainers need to know is those 4 questions asked in the preload template. Let's talk more on phab so others can chime in; I've got this along with a number of other things I wanted to ask you all about.